pkg/apply/filter/inventory-policy-apply-filter.go

```diff
@@ -20,6 +20,7 @@ import (
 // if an object should be applied based on the cluster object's inventory id,
 // the id for the inventory object, and the inventory policy.
 type InventoryPolicyApplyFilter struct {
+	Context   context.Context
```

Comment:
Suggest adding context to the Filter method signature in the interface.

Reply:
done